packages/gestalt/src/ScrollFetch.js

```diff
@@ -32,7 +32,9 @@ export default class ScrollFetch extends React.PureComponent<Props, State> {
     this.setState(this.getScrollState());
   });
 
-  static defaultProps: {};
+  static defaultProps = {
```

Comment:
As it turns out, this code was actually compliant already. `defaultProps` was only deprecated on function components, and this component is a class component. That said, I prefer moving it into a static declaration rather than putting it at the bottom of the file, so I like this change.